docs/usage/local.md

```diff
@@ -7,7 +7,7 @@ To quickly test and run Lodestar we recommend starting a local testnet. We recom
 Run a beacon node, with 8 validators with the following command.
 
 ```bash
-./lodestar dev --genesisValidators 8 --genesisTime 1578787200 --enr.ip 127.0.0.1 --rootDir </path/to/node1> --reset
+./lodestar dev --genesisValidators 8 --genesisTime 1578787200 --startValidators 0:1 --enr.ip 127.0.0.1 --rootDir </path/to/node1> --reset
```

Comment:
It would be better to run with 8 validators in the main example. A network with only 1 validator is weird

Reply:
changes reverted